bin/node/runtime/src/lib.rs

```diff
@@ -1013,7 +1058,7 @@ construct_runtime!(
 );
 
 /// The address format for describing accounts.
-pub type Address = <Indices as StaticLookup>::Source;
+pub type Address = sp_runtime::MultiAddress<AccountId, AccountIndex>;
```

Comment:
Should any updates be applied to Account type Cere's pallets?

Reply:
I haven't found any usage of this type in our pallets. You probably mix it up with AccountId